doc/release-notes/9.0.md

```diff
@@ -0,0 +1,263 @@
+# Release notes for Gluster 9.0
+
+This is a major release that includes a range of features, code improvements and stability fixes as noted below.
+
+A selection of the key features and changes are documented in this page.
+A full list of bugs that have been addressed is included further below.
+
+- [Announcements](#announcements)
+- [Major changes and features](#major-changes-and-features)
+- [Major issues](#major-issues)
+- [Bugs addressed in the release](#bugs-addressed)
+
+## Announcements
+
+1. Releases that receive maintenance updates post release 9 is 8 
+([reference](https://www.gluster.org/release-schedule/))
+
+2. Release 9 will receive maintenance updates around the 30th of every month
```

Comment:
` Release 9 will receive maintenance updates around the 30th of every month` release 8 or 9?

Reply:
In release note of 9, we should be talking about release 9.0 only....So further maintenance releases on 30th of every month. Is there anything I missed?.

Reply:
@msaju Sorry, I got it now